packages/react-reconciler/src/__tests__/ReactDoubleInvokeEvents-test.internal.js

```diff
@@ -312,19 +355,45 @@ describe('ReactDoubleInvokeEvents', () => {
       expect(Scheduler).toHaveYielded(['componentDidMount']);
     }
 
-    ReactNoop.act(() => {
-      ReactNoop.render(<App text={'update'} />);
+    act(() => {
+      renderer.update(<App text={'update'} />);
     });
 
     expect(Scheduler).toHaveYielded(['componentDidUpdate']);
 
-    ReactNoop.act(() => {
-      ReactNoop.render(null);
+    act(() => {
+      renderer.unmount();
```

Comment:
Just curious - why are you switching from render to update/unmount?

Reply:
`renderer.unmount()` is the canonical way to unmount a root using the test renderer. `renderer.update(null)` would have also worked here.

Reply:
So you were just cleaning up the `.render` calls?

Reply:
No. In order to test the changes made (legacy root vs modern root) I needed to use a renderer that supported both. `ReactNoop` did not, so I switched to `react-test-renderer`.

Reply:
Ah! I forgot about that method.

Reply:
Ah got it, thanks for the context!